rllib/core/learner/differentiable_learner.py

```diff
@@ -116,17 +119,21 @@ def __init__(
         self.iterator: MiniBatchRayDataIterator = None
 
     @OverrideToImplementCustomLogic_CallToSuperRecommended
-    def build(self) -> None:
+    def build(self, device: Optional[DeviceType]) -> None:
```

Comment:
Same question as I had on the rayturbo PR:
Why can't we infer the device automatically, like we do in the `(Torch)Learner` class?